lomond/session.py

```diff
@@ -13,10 +13,14 @@
 import socket
 import ssl
 import threading
-import time
 
 from six.moves.urllib.parse import urlparse
 
+try:
+    from monotonic import monotonic as monotonic_time
+except:
```

Comment:
Best to explicitly catch an `ImportError` in the off chance the module has any import time side-effects.